src/scripts/main.js

```diff
@@ -3,6 +3,42 @@
 const people = require('./lib/people');
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+console.log(people);
 
-// write your code here
+const table = document.querySelector('tbody');
+
+for (const person of people) {
+  const someOne = document.createElement('tr');
```

Comment:
```suggestion
  const row = document.createElement('tr');
```

Reply:
Taras, hi! Thank you for review and recommendation. I have done it.